subprojects/performance/templates.gradle

```diff
@@ -80,6 +80,21 @@ tasks.register("createLotsOfTasks", JvmProjectGeneratorTask) {
     numberOfExternalDependencies = 0
 }
 
+tasks.register("createLotsOfDeprecationWarnings", JvmProjectGeneratorTask) {
+    projects = 1
+    sourceFiles = 0
+    subProjectTemplates = ['deprecations']
```

Comment:
You might wanna use a simple Copy task, like I did for the archivePerformanceTest. We don't need any of the advanced project generation stuff here.